remote-api/src/main/java/org/alfresco/rest/framework/webscripts/ResourceWebScriptGet.java

```diff
@@ -57,6 +57,8 @@
 public class ResourceWebScriptGet extends AbstractResourceWebScript implements ParamsExtractor, RecognizedParamsExtractor
 {
     private static Log logger = LogFactory.getLog(ResourceWebScriptGet.class);
+
+    private static final String GET_FOLDERSIZE = "/nodes/{id}/size";
```

Comment:
This seems to be generic Get Class which acts as an Get Interceptor, I suppose we shall not have any business logic related stuff here. That is my understanding.

Reply:
Same as the above comment: All the changes in this class are unnecessary and should be removed, as they have no impact on the actual feature.